Quoridor.Core/Pathfinding/PathFinder.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using HavocAndCry.Quoridor.Core.Models;
+
+namespace HavocAndCry.Quoridor.Core.Pathfinding
+{
+    public class PathFinder
+    {
+        // Finds whether a path from player to his finish row exists using Lee (wave) algorithm
+        public static bool IsPathToExit(Player player, GameField gameField)
```

Comment:
Better rename it to `ExistsPathToExit`, and do we really need static here? Maybe create `IPathFinder` and make this class implements him? It will be more flexible.

Reply:
Of course, i think we need IPathFinder interface here

Reply:
Using IPathFinder interface really seems better, so the question about the static is automatically solved. As for the method's name, I don't like the 'Exists' at the beginning, maybe IsPathToFinishExists is a more accurate naming.

Reply:
Okay, then extract interface and name the method `IsPathToFinishExists`